ExomeDepth/igv_xml_session.py

```diff
@@ -57,10 +57,10 @@ def make_single_igv_file(args, sample_id, statistic, igv_extension, vcf_extensio
     min_axis, mid_axis, max_axis = settings.igv_settings[statistic]  # Get axis values out of settings.py
 
     baf = "../baf/{0}_baf.igv".format(args.sampleid)
-    igv_hc_ratio = "igv_tracks/HC_{0}_{1}_{2}_{3}".format(
+    igv_hc_ratio = "igv_tracks/{1}_{2}_HC_{0}_{3}".format(
         args.refset, args.sampleid, args.runid, igv_extension
     )  # CNV igv session with ratios for HC
-    igv_umcu_ratio = "igv_tracks/UMCU_{0}_{1}_{2}_{3}".format(
+    igv_umcu_ratio = "igv_tracks/{1}_{2}_UMCU_{0}_{3}".format(
         args.refset, args.sampleid, args.runid, igv_extension
     )  # CNV igv session with ratios for UMCU
```

Comment:
This really confusing, just update argument order or f-string / named arguments:

```python
gv_hc_ratio = f"igv_tracks/{args.sampleid}_{args.runid}_HC_{args.refset}_{igv_extension}"
```